src/Session/profile.py

```diff
@@ -35,7 +35,7 @@ def getFoodStorage(self, foodStorage_type, name):
         if foodStorage_type not in self._foodStorages or \
             name not in self._foodStorages[foodStorage_type]:
             return None
-        return self._foodStorages[foodStorage_type][name]
+        return self._foodStorages[foodStorage_type][name] 
```

Comment:
I think you added an extra space here by mistake

Reply:
fixed!